applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/child_image/hci_rpmsg_ZDebug.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_LOG=n
+
+CONFIG_RPMSG_SERVICE=y
+CONFIG_RPMSG_SERVICE_MODE_REMOTE=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=16
```

Comment:
Consider setting it to `1` to be consistent with application configuration requirements. CAF BLE implementation assumes that Bluetooth peripheral has only 1 simultaneous connection.

Reply:
Ok.

Reply:
2 is minimal and default value, cant get lower due to Kconfig dependencies.

Reply:
I would just remove that line.